swagger.json

```diff
@@ -3405,6 +3428,37 @@
                     }
                 }
             }
+        },
+        "AssetLocationResponse": {
+            "type": "object",
```

Comment:
The response is an array of these objects, instead of just 1 object

Reply:
lol good catch